ql/src/test/results/clientpositive/llap/insert1_overwrite_partitions.q.out

```diff
@@ -198,9 +198,7 @@ PREHOOK: type: QUERY
 POSTHOOK: query: EXPLAIN INSERT OVERWRITE TABLE destinTable PARTITION (ds='2011-11-11', hr='11') if not exists
 SELECT one,two FROM sourceTable WHERE ds='2011-11-11' AND hr='12' order by one desc, two desc limit 5
 POSTHOOK: type: QUERY
-STAGE DEPENDENCIES:
```

Comment:
I'm going to back down from this change. It seems there were occasions where this was empty. Let's just keep it as it was for backwards compatibility. If we want to remove them, we can do it in a separate JIRA. Sorry about the confusion.

Reply:
Reverted.